src/core/PWSprefs.h

```diff
@@ -147,7 +147,7 @@ class PWSprefs
     MainToolBarButtons, PasswordFont, TreeListSampleText, PswdSampleText,
     LastUsedKeyboard, VKeyboardFontName, VKSampleText, AltNotesEditor,
     LanguageFile, DefaultSymbols, NotesFont, NotesSampleText, AutotypeTaskDelays,
-    AddEditFont, AddEditSampleText,
+    AddEditFont, AddEditSampleText, AltNotesEditorCmdLineParms,
```

Comment:
I'd much rather have the AltNotesEditor's semantics change s.t. it can contain the params as well as the command name, e.g.,
"Path to editor maybe with spaces .exe" "param1" "param2"

rather than adding a separate preference.